unit_test/solver/network/assets/test_boundary.py

```diff
@@ -0,0 +1,78 @@
+#  Copyright (c) 2023. Deltares & TNO
+#
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Test Boundary entities."""
+import unittest
+from unittest.mock import patch
+from uuid import uuid4
+
+from simulator_core.solver.matrix.core_enum import IndexEnum
+from simulator_core.solver.network.assets.base_asset import BaseAsset
+from simulator_core.solver.network.assets.boundary import BaseBoundary
+from simulator_core.solver.network.assets.node import Node
+
+
+class BaseBoundaryTest(unittest.TestCase):
+    """Testcase for Boundary class."""
+
+    def setUp(self) -> None:
+        """Set up the test case."""
+        # Create a BaseBoundary object
+        self.asset = BaseBoundary(
+            name=uuid4(),
+        )
+        # Create supply, connection_point:0 and return node, connection_point:1
+        self.supply_node = Node(name=uuid4())
+        self.return_node = Node(name=uuid4())
```

Comment:
deze wordt niet gebruikt

Reply:
Removed, ik vraag mij af of hij ook wel checkt of alle nodes zijn verbonden.